### PR TITLE
feat: add spec of mint manager

### PR DESCRIPTION
**Pull request by Pangssu**

### CODE REVIEW

**Comment by 0xHansLee** [comment]
Please rebase to the main first

**Review by 0xHansLee — APPROVED** [review]
LGTM

File: specs/protocol/mint-manager.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+- `SLIDING_WINDOW_BLOCKS`: The period (in blocks) which the mint amount decreases.
+- `INIT_MINT_PER_BLOCK`: The amount minted per block during the first epoch.
+- `DECAYING_FACTOR`: The ratio by which the mint amount decreases from the previous period.
+- `DECAYING_DENOMINATOR`: The denominator of the decaying factor. This value is `10^5`.
+- `FLOOR_UNIT`: The unit for rounding down decimal places. Since the decimal places for `GovernanceToken` are 18,
+  and the mint amount is considered up to 8 decimal places, this value is `10^10`
+
+## Initial Minting
+
+Mints the amount of tokens that should have been minted from the genesis block to the current block.
```
Comment (sm-stack) [inline specs/protocol/mint-manager.md]:
```suggestion
Mints the amount of tokens that should have been minted from the L2 genesis block to the current block.
```

**Review by sm-stack — APPROVED** [review]
LGTM, with a nit

**Review by kangsorang — APPROVED** [review]

**Pull request merged**